src/test_suite/test_suite.ts

```diff
@@ -218,6 +254,26 @@ export class AggregatedResults {
         this.printLatencyStatistics();
     }
 
+    async toJUnitXml() {
```

Comment:
I don't see any need for this to be `async` at the moment... are you future proofing the signature for async/io or something?

Reply:
Good catch, development artifact...originally I would have it write to disk but then i changed my mind and had it return the XML instead